src/nano_contracts/header.ts

```diff
@@ -114,7 +114,7 @@ class NanoContractHeader extends Header {
       array.push(intToBytes(this.script.length, 2));
       array.push(this.script);
     } else {
-      array.push(intToBytes(0, 1));
+      array.push(intToBytes(0, 2));
```

Comment:
suggestion(non-blocking): Add a comment explaining why we're using two bytes to store a zero value here, as it's not intuitive.

Reply:
"Done